def/flow.js

```diff
@@ -164,13 +168,24 @@ def("Identifier")
 def("TypeParameterDeclaration")
   .bases("Node")
   .build("params")
-  .field("params", [def("Identifier")]);
+  .field("params", [def("TypeParameter")]);
 
 def("TypeParameterInstantiation")
   .bases("Node")
   .build("params")
   .field("params", [def("Type")]);
 
+def("TypeParameter")
+  .bases("Type")
+  .build("name", "variance", "bound")
+  .field("name", String)
+  .field("variance",
+         or("plus", "minus", null),
```

Comment:
Yeah, I can see that. Internally in Flow we use a variant type `Plus | Minus`, but that we opted to convert that to "plus" and "minus" instead of "+" and "-" in our JS AST is an accident of history.